pkg/api/validation/register.go

```diff
@@ -1,6 +1,8 @@
 package validation
 
 import (
+	_ "github.com/openshift/origin/pkg/api/install"
```

Comment:
This is some weird setup - there's another panic because another package calls register, instead of expecting it to be registered by init().  It's a later test though...